offgridplanner/static/js/pages/demand_estimation.js

```diff
@@ -500,3 +496,29 @@ function loadDashboard() {
         dashboardSection.classList.add('loading');
     }
 }
+
+
+async function export_demand(file_type) {
```

Comment:
Isn't that cleaner to get the .xlsx file from the view directly? We can change all of those js side file download calls subsequentially